docs/src/use/configure/configuration-files.md

```diff
@@ -1,451 +1,396 @@
+```js
+// eslint.config.js
+import exampleConfigs from "eslint-config-example";
 
-`~/` refers to [the home directory of the current user on your preferred operating system](https://nodejs.org/api/os.html#os_os_homedir). The personal configuration file being referred to here is `~/.eslintrc.*` file, which is currently handled differently than other configuration files.
+export default [
+    ...exampleConfigs,
 
-### How does ESLint find personal configuration files?
+    // your modifications
+    {
+        rules: {
+            "no-unused-vars": "warn"
+        }
+    }
+];
+```
+
+Please refer to the documentation for the shareable configuration package you're using to determine whether it is exporting an object or an array.
 
-If `eslint` could not find any configuration file in the project, `eslint` loads `~/.eslintrc.*` file.
+## Configuration File Resolution
 
-If `eslint` could find configuration files in the project, `eslint` ignores `~/.eslintrc.*` file even if it's in an ancestor directory of the project directory.
+When ESLint is run on the command line, it first checks the current working directory for `eslint.config.js`. If the file is not found, it looks to the next parent directory for the file. This search continues until either the file is found or the root directory is reached.
 
-### How do personal configuration files behave?
+You can prevent this search for `eslint.config.js` by using the `-c` or `--config` option on the command line to specify an alternate configuration file, such as:
 
-`~/.eslintrc.*` files behave similarly to regular configuration files, with some exceptions:
+```shell
+npx eslint --config some-other-file.js **/*.js
+```
 
-`~/.eslintrc.*` files load shareable configs and custom parsers from `~/node_modules/` – similarly to `require()` – in the user's home directory. Please note that it doesn't load global-installed packages.
+In this case, ESLint does not search for `eslint.config.js` and instead uses `some-other-file.js`.
 
-`~/.eslintrc.*` files load plugins from `$CWD/node_modules` by default in order to identify plugins uniquely. If you want to use plugins with `~/.eslintrc.*` files, plugins must be installed locally per project. Alternatively, you can use the [`--resolve-plugins-relative-to` CLI option](../command-line-interface#--resolve-plugins-relative-to) to change the location from which ESLint loads plugins.
+By default, all configuration objects are applied to all files ending with `.js`, `.cjs`, and `.mjs`. Sometimes, however, a more fine-grained application of configuration settings is necessary. For example, what if you'd like to apply a different configuration to different files in the same directory?
```

Comment:
This doesn't seem related to Configuration File Resolution.

Reply:
Hmm yeah, looks like a copy-paste error.